index.d.ts

```diff
@@ -3,4 +3,5 @@
  * @param value Input value to be interpreted as DNI
  */
 import FieldValidationResult from 'lc-form-validation';
+export const VALIDATION_TYPE : string;
```

Comment:
I would add `export declare const` in both entities, the constant and the function:
```
export declare const VALIDATION_TYPE: string;
export declare function validateDNI(value: any): FieldValidationResult;
```